evalcache.plugin.zsh

```diff
@@ -7,7 +7,17 @@
 export ZSH_EVALCACHE_DIR=${ZSH_EVALCACHE_DIR:-"$HOME/.zsh-evalcache"}
 
 function _evalcache () {
-  local cacheFile="$ZSH_EVALCACHE_DIR/init-${1##*/}.sh"
+  local cmdHash="nohash"
+
+  # Use md5 on MacOS
```

Comment:
Recommend removing these comments, they obscure the logic a bit (since we aren't actually checking OS, but rather command presence).